step-automation-packages/step-automation-packages-manage/pom.xml

```diff
@@ -8,19 +8,24 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>step-automation-packages-reader</artifactId>
-    <packaging>jar</packaging>
-
     <parent>
-        <groupId>ch.exense.step</groupId>
         <artifactId>step-automation-packages</artifactId>
+        <groupId>ch.exense.step</groupId>
         <version>0.0.0-SNAPSHOT</version>
-        <relativePath>../../step-automation-packages</relativePath>
     </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>step-automation-packages-manage</artifactId>
+
+    <properties>
+    </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
@jeromecomte 
The dependency to the controller is also required, because we have Services class in the same module. To remove this dependency in the future we should both stop using ExecutionScheduler and move Services to the separate module...